common/membership/rp_cluster_test.go

```diff
@@ -67,14 +71,18 @@ func NewTestRingpopCluster(ringPopApp string, size int, ipAddr string, seed stri
 			logger.Error("Failed to create tchannel", tag.Error(err))
 			return nil
 		}
-		listenAddr := ipAddr + ":0"
+		listenAddr := listenIpAddr + ":0"
 		err = cluster.channels[i].ListenAndServe(listenAddr)
 		if err != nil {
 			logger.Error("tchannel listen failed", tag.Error(err))
 			return nil
 		}
 		cluster.hostUUIDs[i] = uuid.New()
-		cluster.hostAddrs[i] = cluster.channels[i].PeerInfo().HostPort
+		cluster.hostAddrs[i], err = ExternalAddressResolver(cluster.channels[i].PeerInfo(), &broadcastIP)
+		if err != nil {
+			logger.Error("tchannel hostport parse", tag.Error(err))
```

Comment:
Error message should mention action we've just tried to perform. There is no "parse" in the call above. I like messages above: `Failed to ...`.